Sources/KSCrashRecordingCore/KSThread.c

```diff
@@ -52,6 +68,30 @@ bool ksthread_getThreadName(const KSThread thread, char *const buffer, int bufLe
     return pthread_getname_np(pthread, buffer, (unsigned)bufLength) == 0;
 }
 
+int ksthread_getThreadState(const KSThread thread)
+{
+    integer_t infoBuffer[THREAD_BASIC_INFO_COUNT] = { 0 };
+    thread_basic_info_t info = infoBuffer;
```

Comment:
I believe pod spec lint problem is here:
` incompatible pointer types initializing 'thread_basic_info_t' with an expression of type 'integer_t[10]’`